events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -46,8 +47,8 @@
     // END UTILITY FUNCTIONS
     // *************************************
 
-    /* CREATE A GEM: Checks that gem does not already exist, then calculates position over avatar's head and 
-    creates a gem there */
+    // Checks that a Bingo gem does not already exist, then calculates the correct position
+    // over the avatar's head and creates a gem there
     var playingMarker;
     function createGem() {
         if (playingMarker) {
```

Comment:
yes: i'd definitely like to make it a habit. so i'll change it here! only reason it wasn't that way here is because i missed it during _my_ comb-through of this older code.